lte/gateway/c/core/oai/tasks/s1ap/s1ap_state.cpp

```diff
@@ -218,6 +218,7 @@ void remove_ues_without_imsi_from_ue_id_coll() {
 
     // for each ue comp_s1ap_id in eNB->ue_id_coll, check if it has an S1ap
     // ue_context, if not delete it
+    mme_ue_id_no_imsi_list.clear();
```

Comment:
There is already a PR with the changes https://github.com/magma/magma/pull/14672 right?

Reply:
The PR, 14672 was raised on branch v1.8. But these changes are also required on master. So I thought, let these changes get merged on to master and back port this PR on branch v1.8